doc/langref/destructuring_to_existing.zig

```diff
@@ -0,0 +1,17 @@
+const print = @import("std").debug.print;
+
+pub fn main() void {
+    var x: u32 = undefined;
+    var y: u32 = undefined;
+    var z: u32 = undefined;
+
+    const tuple = .{ 1, 2, 3 };
+
+    x, y, z = tuple;
```

Comment:
Maybe add usage of `_` to the example to demonstrate throwing away (multiple) elements you're not interested in?

Reply:
Good idea.